styles/style.css

```diff
@@ -92,11 +98,22 @@ p {
   border-left-color: var(--orange-dark);
 }
 
-
 /* Buttons */
 
-  /* INSERT BUTTON STYLES HERE */
+.dark-button {
+  background-color: var(--orange-dark);
+  color: var(--white);
+  border-radius: 0.5rem;
+  padding: 0.5rem;
+}
 
+.light-button {
+  background-color: var(--white);
+  color: var(--orange-dark);
+  border-radius: 0.5rem;
```

Comment:
Looks like we use the same settings for border radius and padding for both button types. Can we reduce code duplication?